prog/probe.go

```diff
@@ -40,7 +40,10 @@ const (
 	defaultServiceHost = "https://cloud.weave.works:443"
 )
 
-var pluginAPIVersion = "1"
+var (
+	pluginAPIVersion = "1"
+	dockerEndpoint   = "unix:///var/run/docker.sock"
```

Comment:
Maybe, not part in this PR though. I was nice enough to refactor it a bit :)

I have created https://github.com/weaveworks/scope/issues/1975